templates/index.html

```diff
@@ -9,9 +9,11 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
```

Comment:
You'll need to provide a link to the other pages here (as a new user will not know that there are hidden urls for creating new todos)

Reply:
Done! thanks.